pkg/cmd/uninstall/uninstall.go

```diff
@@ -245,3 +252,19 @@ func (o *UninstallOptions) DeleteReleaseIfPresent(ns string, releaseName string,
 	}
 	return errors
 }
+
+func (o *UninstallOptions) setPVCFinalizerToNull(ns string, releaseName string) error {
+	client, err := o.KubeClient()
+	if err != nil {
+		return errors.Wrap(err, "getting the kube client")
+	}
+
+	list, _ := client.CoreV1().PersistentVolumeClaims(ns).List(meta_v1.ListOptions{})
```

Comment:
I think we should not ignore this error, there may be a problem with the connection there. The list will be empty instead of nil so there won't be a panic, but I think we should handle all errors.